marklogic-data-hub-central/ui/src/components/common/hc-datetime-picker/hc-datetime-picker.module.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../../HCTheme.scss";
+
+.pickerContainer {
+    background-color: white;
+    border-radius: 4px;
+    border: 1px solid #d9d9d9;
+    position: relative;
+    max-width: 265px;
+    
+    &:focus-visible {
+        border: none;
+        outline: none;
+    }
+}
+.input {
+    background-color: transparent;
+    border: none;
+    color: $text-color;
+    padding: 4px 11px;
+    width: 100%;
+    font-size: $font-size-base;
+    text-align: center;
+    
+    &:focus-visible {
+        border: #808cbd 1px solid;
+        border-radius: 4px;
+        outline: none;
+    }
+}
+.calendarIcon {
+    position: absolute;
+    top: 8px;
+    right: 8px;
+    color: $light;
+    background-color: white;
+    font-size: 12px;

```

Comment:
Can we please add the following to .calendarIcon since it is clickable:

cursor: pointer;